src/task-definitions.js

```diff
@@ -0,0 +1,347 @@
+        e.returnValue = false
+      }
+      const cellBelowId = getCellBelowSelectedId()
+      if (cellBelowId !== null) { dispatcher.selectCell(cellBelowId, true) }
+    }
+  },
+})
+
+
+TASKS.AddCellAbove = new NotebookTask({
+  title: 'Add Cell Above',
+  keybindings: ['a'],
+  displayKeybinding: 'a',
+
+  callback() {
+    dispatcher.insertCell('javascript', 'above')
+    dispatcher.selectCell(getCellAboveSelectedId(), true)
+  },
+  keybindingCallback() {
+    if (isCommandMode()) {
+      dispatcher.insertCell('javascript', 'above')
+      dispatcher.selectCell(getCellAboveSelectedId(), true)
+    }
+  },
+})
+
+TASKS.AddCellBelow = new NotebookTask({
+  title: 'Add Cell Below',
+  keybindings: ['b'],
+  displayKeybinding: 'b',
+
+  callback() {
+    dispatcher.insertCell('javascript', 'below')
+    dispatcher.selectCell(getCellBelowSelectedId(), true)
+  },
+  keybindingCallback() {
+    if (isCommandMode()) {
+      dispatcher.insertCell('javascript', 'below')
+      dispatcher.selectCell(getCellBelowSelectedId(), true)
+    }
+  },
+})
+
+TASKS.DeleteCell = new NotebookTask({
+  title: 'Delete Cell',
+  keybindings: ['shift+del', 'shift+backspace'],
+  displayKeybinding: '\u21E7 \u232b',
+
+  callback() { dispatcher.deleteCell() },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.deleteCell()
+  },
+})
+
+TASKS.ChangeToJavascriptCell = new NotebookTask({
+  title: 'Change to Javascript',
+  keybindings: ['j'],
+  displayKeybinding: 'J',
+
+  callback() {
+    dispatcher.changeCellType('javascript')
+  },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.changeCellType('javascript')
+  },
+})
+
+TASKS.ChangeToMarkdownCell = new NotebookTask({
+  title: 'Change to Markdown',
+  keybindings: ['m'],
+  displayKeybinding: 'M',
+
+  callback() {
+    dispatcher.changeCellType('markdown')
+  },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.changeCellType('markdown')
+  },
+})
+
+TASKS.ChangeToExternalResourceCell = new NotebookTask({
+  title: 'Change to External Resource',
+  keybindings: ['e'],
+  displayKeybinding: 'E',
+
+  callback() {
+    dispatcher.changeCellType('external dependencies')
+  },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.changeCellType('external dependencies')
+  },
+})
+
+TASKS.ChangeToRawCell = new NotebookTask({
+  title: 'Change to Raw',
+  keybindings: ['r'],
+  displayKeybinding: 'R',
+
+  callback() {
+    dispatcher.changeCellType('raw')
+  },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.changeCellType('raw')
+  },
+})
+
+TASKS.ChangeToCSSCell = new NotebookTask({
+  title: 'Change to CSS',
+  keybindings: ['c'],
+  displayKeybinding: 'C',
+
+  callback() {
+    dispatcher.changeCellType('css')
+  },
+  keybindingCallback() {
+    if (isCommandMode()) dispatcher.changeCellType('css')
+  },
+})
+
+TASKS.EditMode = new NotebookTask({
+  title: 'Change to Edit Mode',
+  keybindings: ['enter', 'return'],
+  displayKeybinding: '⎋',
```

Comment:
i don't recognize this symbol: '⎋'. Can we just use the word `enter` written out? i think it will be a little clearer to more people.

In Jupyter's keybd shortcut display (and other places), they wrap all the shorcut in little box. i think it's very clear and looks fine:

![image](https://user-images.githubusercontent.com/2380975/36467037-b5b79c64-1691-11e8-9cd3-d7fa43fc953e.png)

Reply:
This is a mistake - that's technically the `esc` key not the `return` key, but honestly `Esc` explains everything and this is evidence enough that the char for it is not supported widely.